src/core/hle/D3D8/Direct3D9/FixedFunctionPixelShader.hlsl

```diff
@@ -245,6 +245,24 @@ TextureArgs ExecuteTextureStage(
 
 float4 main(const PS_INPUT input) : COLOR {
 
+// Calculate the fog factor
+	// Some of this might be better done in the pixel shader?
+		float fogFactor;
```

Comment:
Incorrect indentation

Reply:
fixed

Reply:
I still see space vs tab differences compared to surrounding lines?